dts/bindings/display/ilitek,ili9341.yaml

```diff
@@ -72,7 +72,7 @@ properties:
 
     disctrl:
       type: uint8-array
-      default: [0x0a, 0x82, 0x27]
+      default: [0x0a, 0x82, 0x27, 04]
```

Comment:
Going mental: better would have been 0x04 to align notation with the rest

Reply:
Ah no, why did I write it like that? Fixed now.